tests/integration/test_pairwise_significance.py

```diff
@@ -8,12 +8,151 @@
 
 from cr.cube.cube import Cube
 
-from ..fixtures import CR
+from ..fixtures import CR, SM
 
 
 class TestStandardizedResiduals(TestCase):
     """Test cr.cube implementation of column family pairwise comparisons"""
 
+    def test_pairwise_measures_scale_means_nps_type(self):
```

Comment:
@malecki this test is the most significant. In here we compare col 0 with others and col 1 with others using the fixture (facebook app vs ages)